rasa/core/policies/ted_policy.py

```diff
@@ -819,13 +819,20 @@ def _encode_features_per_attribute(
         Returns:
             A tensor combining  all features for `attribute`
         """
-        attribute_mask = tf_batch_data[attribute][MASK][0]
+        # attribute_mask = tf_batch_data[attribute][MASK][0]
 
         if attribute in SEQUENCE_FEATURES_TO_ENCODE:
             _sequence_lengths = tf.cast(
```

Comment:
should we add a comment that sequence length contains 0 length if no features for this attribute are present?